shell/platform/embedder/embedder.h

```diff
@@ -715,6 +766,19 @@ FlutterEngineResult FlutterEnginePostRenderThreadTask(FlutterEngine engine,
                                                       VoidCallback callback,
                                                       void* callback_data);
 
+// Get the current time in nanoseconds from the clock used by the flutter
```

Comment:
s/Get/Gets/ (per Google commenting style for C++)

Reply:
Done.